terraform/modules/vault/acm.tf

```
@@ -0,0 +1,34 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2014-2019 Avant, Sean Lingren
+
+resource "aws_acm_certificate" "acm" {
+  count             = "${ var.route53_enabled ? 1 : 0 }"
+  domain_name       = "${ var.route53_domain_name }"
+  validation_method = "DNS"
+
+  tags = "${ merge(
+    map("Name", "${ var.name_prefix }_private"),
```

I don't think we need `_private` in the `Name` tag here, the certificate could be public or private.
```suggestion
    map("Name", "${ var.name_prefix }"),
```

---

Done